metaci/repository/migrations/0009_default_implementation_steps.py

```diff
@@ -13,6 +13,6 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name="repository",
             name="default_implementation_steps",
-            field=models.JSONField(default=[], null=True),
+            field=models.JSONField(default=list, null=True),
```

Comment:
Was this migration already run in production? If so then you need to create a new migration rather than changing the existing one.